### ISSUE TITLE
Format Document doesn't trigger fix all auto-fixable problems

### ISSUE THREAD

**Issue by hamzahamidi** [microsoft/vscode-eslint#783]
Format document or alt + shift +f used to fix all auto-fixable problems. Is it because I use ESLINT 6?
Here's a minimal package.json:

```
{
  "version": "1.0.0",
  "main": "index.js",
  "scripts": {
    "test": "npm test",
    "build": "webpack --watch --config webpack.config.js --mode development",
    "build:prod": "webpack --config webpack.config.js --mode production",
    "lint": "./node_modules/.bin/eslint --fix src/* test/*"
  },
  "pre-commit": [
    "lint"
  ],
  "license": "MIT",
  "devDependencies": {
    "eslint": "^6.5.1",
    "webpack": "^4.41.2",
    "webpack-cli": "^3.3.9",
    "jest": "^24.9.0"
  }
}
```

**Comment by dbaeumer** [comment]
@hamzahamidi can you provide me with a GitHub repository I can clone that demos what you are experiencing.

**Comment by hamzahamidi** [comment]
Thank you for your reply @dbaeumer Here's the [repository](https://github.com/hamzahamidi/fetch-layer).

**Comment by dbaeumer** [comment]
Actually the ESLint extension is not contributing a formatter. So Format Document never used eslint fixes. So I am not sure what formatter you had configured before to make this work. There is: https://github.com/microsoft/vscode-eslint/issues/417 to enable this.

I will close the issue.

If you use prettier you might be interested in https://github.com/microsoft/vscode-eslint/issues/696 which describes problem with prettier and ESLint 6.x usage.
